README.md

```diff
@@ -1 +1,126 @@
-# aelf.graphql
\ No newline at end of file
+# AElf GraphQL
+
+An GraphQL module for use in ABP and Orleans framework.
+
+- [About The Project](#about-the-project)
+- [Getting Started](#getting-started)
+  - [Installation](#installation)
+  - [Configuration](#configuration)
+  - [Setup](#setup)
+    - [Orleans](#orleans)
+- [Examples](#examples)
+- [Contributing](#contributing)
+- [License](#license)
+
+## About The Project
+
+This project is a module for ABP and Orleans framework to use aelf graphql. It provides a simple and fast way to integrate aelf graphql in ABP and Orleans framework.
+
+## Getting Started
+
+### Installation
```

Comment:
Please check https://github.com/AElfProject/aelf-dapp-factory/ for a command to automatically initialise your repo with the required things such as .github workflows.